src/test/java/net/snowflake/client/jdbc/telemetryOOB/TelemetryServiceIT.java

```diff
@@ -248,14 +283,38 @@ public void testSnowflakeSQLLoggedExceptionOOBTelemetry()
   @Test
   public void testSnowflakeSQLLoggedExceptionIBTelemetry() throws SQLException {
     // make a connection to initialize telemetry instance
-    Connection con = getConnection();
-    int fakeErrorCode = 27;
-    try {
-      generateDummyException(fakeErrorCode, con.unwrap(SnowflakeConnectionV1.class).getSfSession());
-      fail();
-    } catch (SnowflakeSQLLoggedException e) {
-      // The error response has the same code as the fakeErrorCode
-      assertThat("Communication error", e.getErrorCode(), equalTo(fakeErrorCode));
+    try (Connection con = getConnection()) {
+      int fakeErrorCode = 27;
+      try {
+        generateDummyException(
+            fakeErrorCode, con.unwrap(SnowflakeConnectionV1.class).getSfSession());
+        fail();
+      } catch (SnowflakeSQLLoggedException e) {
+        // The error response has the same code as the fakeErrorCode
+        assertThat("Communication error", e.getErrorCode(), equalTo(fakeErrorCode));
+      }
+    }
+  }
+
+  /**
+   * Test case for checking telemetry message for SnowflakeFeatureNotSupporteExceptions. In-band
+   * telemetry should be used. After running test, check for telemetry message in client_telemetry_v
+   * table.
+   *
+   * <p>Test is ignored because since responses are immediately sent asynchronously (before
+   * execution stops due to sqlexception being thrown) instead of being stored for awhile, it's
+   * difficult to pull out the telemetry logs and look at them before they're sent.
+   *
+   * @throws SQLException
+   */
+  // @Ignore
```

Comment:
can add a comment why you are skipping? If this test is not required, can you remove this?

Reply:
Are you planning to enable this in the future?

Reply:
I had it ignored because I thought it was more useful as a manual test where the user then goes and checks the telemetry message in the table. However, I suppose it makes sense to run it on Jenkins as well because if there is a telemetry error, an exception should be thrown. I'll add it back in.